packages/vsce/src/utils/profileUtils.ts

```diff
@@ -55,3 +55,30 @@ export function getIconOpen(open: boolean = true) {
     `folder-${open ? "open" : "closed"}-dark.svg`,
     `folder-${open ? "open" : "closed"}-light.svg`);
 }
+
+export function getIconByStatus(resourceType: string, resourceTreeItem: any) {
```

Comment:
A thought - and I know I've added icon-related methods to this file recently - but should we consider creating a new icons file under utils for this as well as the other icon methods (`getIconOpen`, `getIconPathInResources`, now `getIconByStatus`)?? I don't think they relate to profiles, and now we have more methods, perhaps you could make it's own file?? 

I'm thinking `vsce/src/utils/iconUtils.ts` or something similar. What do you think @almightyrush ??